### PR TITLE
Extend SVN type to carry a min acceptable value

### LINKED ISSUE #62: Min SVN
There is a use case for a vendor specifying a maximum SVN value. If the maxsvn is reached, and if the reported svn in evidence is greater than maxsvn then the verifier should fail evidence matching.

If the endorsed svn is greater than maxsvn then that shouldn't happen and is a failure case. 

Th If the reported svn greater than the endorsed svn but less than maxsvn, then possibly there was an update that bumped the svn but didn't update the endorsement tag/manifest. Possibly, a policy is used to determine if svn is acceptable? 

The schema should nevertheless support a maxsvn along with current svn.

### PR DESCRIPTION
**Pull request by yogeshbdeshpande**
Fixes #62

### CODE REVIEW

File: concise-mid-tag.cddl [hunk 1]
```diff
@@ -158,7 +158,12 @@ version-map = {
 version-type = text .default '0.0.0'
 ; version-scheme is defined in CoSWID
 
-svn-type = int
+eq-svn = int
+min-svn = int
+tagged-eq-svn = #6.1000(eq-svn)
+tagged-min-svn = #6.1001(min-svn)
```
Comment (thomas-fossati) [inline concise-mid-tag.cddl:164]: `eq-svn` and `min-svn` are throw-away types
```suggestion
tagged-eq-svn = #6.1000(int)
tagged-min-svn = #6.1001(int)
```

File: concise-mid-tag.cddl [hunk 2]
```diff
@@ -158,7 +158,12 @@ version-map = {
...
+svn-type-choice = tagged-eq-svn / tagged-min-svn
```
Comment (thomas-fossati) [inline concise-mid-tag.cddl:165]: type choices need a `$` prepended (see TYPOGRAPHY.md)

Comment (thomas-fossati) [reply concise-mid-tag.cddl:165]: where is this used?

File: concise-mid-tag.cddl [hunk 3]
```diff
@@ -139,7 +139,7 @@ tagged-impl-id-type = #6.47115(impl-id-type)
 ; see https://github.com/ietf-rats/ietf-corim-cddl/issues/72
 measurement-map = non-empty<{
   ? comid.ver => version-map
-  ? comid.svn => svn-type
+  ? comid.svn => svn-type-choice
```
Comment (thomas-fossati) [inline concise-mid-tag.cddl:142]:
```suggestion
  ? comid.svn => $svn-type-choice
```

File: concise-mid-tag.cddl (hunk 2 above)
Comment (thomas-fossati) [reply concise-mid-tag.cddl:165]:
```suggestion
$svn-type-choice = tagged-eq-svn / tagged-min-svn
```

**Review by nedmsmith — APPROVED** [review]
eq-svn could be simply 'svn' but this is OK too.

**Review by nedmsmith — APPROVED** [review]

**Review by thomas-fossati — CHANGES_REQUESTED** [review]
I have left a few suggestions

File: concise-mid-tag.cddl (hunk 3 above)
Comment (yogeshbdeshpande) [reply concise-mid-tag.cddl:142]: OK,  I will make the change and perhaps few other places also to make it look consistent!

File: concise-mid-tag.cddl (hunk 1 above)
Comment (yogeshbdeshpande) [reply concise-mid-tag.cddl:164]: The way I have represented looks more cleaner  and logically flowing to me, so would prefer to retain the same!

Comment (yogeshbdeshpande) [reply concise-mid-tag.cddl:164]: Also this is in-line with other types as in line 119..

**Comment by yogeshbdeshpande** [comment]
> eq-svn could be simply 'svn' but this is OK too.

Yes Agree: Changes done!
Thanks!

File: concise-mid-tag.cddl (hunk 1 above)
Comment (thomas-fossati) [reply concise-mid-tag.cddl]: > [..] looks more cleaner and logically flowing

I disagree, let's not add any unnecessary clutter please.

**Pull request merged**